src/tzdata/compile.jl

```diff
@@ -508,90 +539,101 @@ function resolve!(zone_name::AbstractString, zoneset::ZoneDict, ruleset::RuleDic
     end
 end
 
-function resolve(zoneset::ZoneDict, ruleset::RuleDict; max_year::Integer=MAX_YEAR, debug=false)
-    ordered = OrderedRuleDict()
-    time_zones = Dict{AbstractString,TimeZone}()
 
-    for zone_name in keys(zoneset)
-        tz = resolve!(zone_name, zoneset, ruleset, ordered; max_year=max_year, debug=debug)
-        time_zones[zone_name] = tz
-    end
+function load!(tz_source::TZSource, io::IO)
+    zones = tz_source.zones
+    rules = tz_source.rules
+    links = tz_source.links
 
-    return time_zones
-end
+    local kind
+    local name
 
-function resolve(zone_name::AbstractString, zoneset::ZoneDict, ruleset::RuleDict; max_year::Integer=MAX_YEAR, debug=false)
-    ordered = OrderedRuleDict()
-    return resolve!(zone_name, zoneset, ruleset, ordered; max_year=max_year, debug=debug)
-end
+    # For the intial pass we'll collect the zone and rule lines.
+    for line in eachline(io)
+        # Lines that start with whitespace can be considered a "continuation line" which
+        # means the last found kind/name should persist.
+        persist = occursin(r"^\s", line)
```

Comment:
It looks like this is only used once, so perhaps the comment could be moved atop the conditional where this is used and the condition could instead be `!occursin(r"^\s", line)`.

Reply:
The `strip` line after this will unfortunately eliminate the space we need to check this.

Reply:
Oh, good call!